inst/string.m

```diff
@@ -1276,3 +1284,31 @@ function disp (this)
     endif
   endfor
 endfunction
+
+## Test string constructor
+%!test
+%! str = string (["a";"b";"c"]);
+%! assert (cell (str), {"a";"b";"c"})
+%!test
+%! str = string ({"a";"b";"c"});
+%! assert (cell (str), {"a";"b";"c"})
+%!test
+%! str = string ({"a";"";"c"});
+%! tfM = ismissing (str);
```

Comment:
I don't think this is right. Blank charvecs in char or cellstr input should convert to nonmissing blank strings, not missing strings. So I think str(2) here should be `""`, not missing.